st2common/st2common/models/api/rule.py

```diff
@@ -105,6 +164,7 @@ class RuleAPI(BaseAPI):
             'description': {
                 'type': 'string'
             },
+            'type': RuleTypeSpec.schema,
```

Comment:
Hm, looks like this `type.ref` attribute is required and we don't specify a default value in ` RuleTypeSpec`.

Shouldn't we specify a default value of `standard` there otherwise the creation will fail if this attribute is not provided?